packages/unmock-core/src/generator.ts

```diff
@@ -152,10 +152,17 @@ const generateMockFromTemplate = (
   // First iteration simply parses these and returns the updated schema
   const resolvedTemplate = jsf.generate(template);
   jsf.reset();
+  // After one-pass resolving we might have new parameters to resolve.
```

Comment:
For this type of try/catch pattern, it would be easier to read if it used `Maybe` from `fp-ts` or a similar type of structure so that `body` could be a `const`.

Reply:
Sure! I'm not sure it's more readable though, I ended up with:
```typescript
const body = JSON.stringify(
  getOrElse(() => resolvedTemplate)(
    tryCatch(() => jsf.generate(resolvedTemplate)),
  ),
);```

Reply:
Decided to instead implement an abstraction a-la:
```typescript
const tryCatch = (v: any, f: (v: any) => any) => {
  try {
    return f(v);
  } catch {
    return v;
};```